dataactcore/models/domainModels.py

```diff
@@ -2,7 +2,8 @@
 
 import sqlalchemy as sa
 
-from sqlalchemy import Column, Date, ForeignKey, Index, Integer, Numeric, Text, Float, UniqueConstraint, Boolean
+from sqlalchemy import Column, Date, DateTime, ForeignKey, Index, Integer, Numeric
+from sqlalchemy import Text, Float, UniqueConstraint, Boolean
```

Comment:
Why were these split up? Shouldn't they all be in the same line if they're from the same place?